src/ising.jl

```diff
@@ -48,6 +48,13 @@ function energy(fg::MetaDiGraph, edge::Edge)
     for (j, η) ∈ enumerate(vec(wSp))
         en[:, j] = energy.(vec(vSp), Ref(edge.J), Ref(η))
     end
+    #=
+    for (j, η) ∈ enumerate(vec(wSp))
```

Comment:
Will we need this? If not, we should remove this code.

Reply:
done